notification/src/handler/notification/notification.handler.js

```diff
@@ -6,6 +6,25 @@ const { getNotificationForTracking } = require('../../utils/commons')
 const ctp = require('../../utils/ctp')
 const mainLogger = require('../../utils/logger').getLogger()
 
+class CommercetoolsError extends Error {
+  constructor({ stack, message, statusCode }) {
+    super()
+    this.stack = stack
+    this.message = message
+
+    /*
+      We do not want to block notifications coming by Adyen.
+      So with this error it will be accepted by notification module by default.
+
+      Only we do not accept for statusCode === 0 (NetworkError) or 5xx errors like 500.
+      Note that, our client configured to do retries for statusCode=503 cases.
+      So when isRecoverable assigned to true the notification will not respond with accepted but with 500 status code.
+    */
+    this.isRecoverable =
+      statusCode !== undefined ? statusCode === 0 && statusCode >= 500 : false
```

Comment:
> statusCode === 0

Why statusCode can be 0? It is either not set due to network error or is defined, right?

Maybe something like that:

`this.isRecoverable = (statusCode !== undefined && statusCode > 399 && statusCode !== 409 && statusCode < 500 ) ? false : true`

Reply:
statusCode is sth set on the SDK, but you are right, I updated the statement (also 409 is added as recoverable)
https://github.com/commercetools/commercetools-adyen-integration/pull/629/commits/4cb04af997f9e9d4b57c48e8b310bac303f97ba4